apps/onboarding/src/session/session.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common'
+import { v4 as uuidv4 } from 'uuid'
+import { Session } from './session.entity'
+import { SessionRepository } from './session.repository'
+
+
+@Injectable()
+export class SessionService {
+
+  constructor(private readonly sessionRepository: SessionRepository,) {}
+
+  async createSession(externalAccount: string): Promise<Session> {
+    const session = Session.of({
+      id: uuidv4(),
+      externalAccount: externalAccount,
+      createdAt: new Date(Date.now()).toISOString(),
+      expiredAt: new Date(Date.now()+ 3600).toISOString(),
+      completedAttestations: 0,
+      requestedAttestations: 0,
+      meta: {"metadata": "TBD"},
+      completedAt: new Date(Date.now() + 36000).toISOString()
```

Comment:
It's unclear why we're setting the `completedAt` here. Is this just a placeholder?

Reply:
I was not sure why we need this field in the session, it was in the specification. But if we don't need it we could remove it